Platform/Shared/VMShareFileModifier.swift

```diff
@@ -24,7 +24,7 @@ struct VMShareItemModifier: ViewModifier {
     #if os(macOS)
     func body(content: Content) -> some View {
         ZStack {
-            SharingsPicker(isPresented: $isPresented, sharingItems: items())
+            SavePanel(isPresented: $isPresented, shareItem: items()[0]) // Only ever called with 1 item
```

Comment:
Can you change this to use a single value instead of an Array?

Reply:
The other part of this (iOS and `UIActivityViewController`) needs an array. It's just that I didn't want to have handling for an array of items on macOS since the code never calls it with more than 1.

Reply:
That makes sense. It would be good to mention this in a code comment.

Reply:
@conath Err, the comment is right on line 27